src/api/http.rs

```diff
@@ -139,19 +148,31 @@ fn not_acceptable_backend() -> Status {
     status
 }
 
-fn find_group<'a>(bob: &'a State<BobServer>, vdisk_id: u32) -> Result<&'a PearlGroup, StatusExt> {
+// !notice: only finds normal group
+fn find_group(bob: &State<BobServer>, vdisk_id: u32) -> Result<PearlGroup, StatusExt> {
     let backend = bob.grinder().backend().inner();
     debug!("get backend: OK");
-    let groups = backend.vdisks_groups().ok_or_else(not_acceptable_backend)?;
+    let (dcs, _) = backend
+        .disk_controllers()
+        .ok_or_else(not_acceptable_backend)?;
     debug!("get vdisks groups: OK");
-    groups
+    let needed_dc = dcs
         .iter()
-        .find(|group| group.vdisk_id() == vdisk_id)
+        .find(|dc| dc.vdisks().iter().find(|&&vd| vd == vdisk_id).is_some())
```

Comment:
I think it's this way to find needed dc is more clear:
```
.flat_map(|dc| dc.vdisks())
.find(|&&vd| vd == vdisk_id)
```

Reply:
That's not what I actually want. When you do `flat_map` you start to iterate through vdisks ids and `find` method is working with iterator through this ids (so it'll return id), but I need `DiskController`